File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletInvalidQueryException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when attempting to use a wallet with the wrong pool.
    /// </summary>
    public class WalletInvalidQueryException : IndyException
    {
        const string message = "The wallet query provided was invalid.";

        /// <summary>
        /// Initializes a new WalletInvalidQueryException.
        /// </summary>
        internal WalletInvalidQueryException() : base(message, (int)ErrorCode.WalletQueryError)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/wallet/WalletInvalidQueryException.java
package org.hyperledger.indy.sdk.wallet;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when provided wallet query is invalid.
 */
public class WalletInvalidQueryException extends IndyException
{
	private static final long serialVersionUID = 667964860056778208L;
	private final static String message = "Wallet query is invalid.";

	/**
	 * Initializes a new WalletInvalidQueryException.
	 */
	public WalletInvalidQueryException()
	{
		super(message, ErrorCode.WalletQueryError.value());
	}
}
